app/controllers/employees_controller.rb

```diff
@@ -6,6 +6,12 @@ def new
   def create
     @employee = Employee.new(employee_params)
 
+    unless EmployeeFinder.new.employee_exists?(@employee.slack_username)
+      flash.now[:error] = "\"#{@employee.slack_username}\" is not a valid employee (that username was not found)."
+      render action: :new
+      return
```

Comment:
In this case we do though, without it the controller action was continuing to flow, causing it to add the invalid employee anyway!  Unless (pun not intended, haha!) there's another/better way to immediately stop and return?